src/components/Events/ConferenceCard.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import styled from 'styled-components'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
+import Image from '../Common/Image'
+
+import Anchor from '../Common/Anchor'
+
+import { Link } from 'gatsby'
+
+import { Row, Col } from '../grid'
+import StyledLink from '../Common/StyledLink'
+import { CardTitle, BodyPrimary } from '../Typography'
+
+const BlurbWrapper = styled.div`
+  ${breakpoint('tablet')`
+    padding-top: ${remcalc(24)};
+  `};
+`
+
+const StatusWrapper = styled.div`
+  padding-top: ${remcalc(12)};
+  padding-bottom: ${remcalc(12)};
+
+  ${breakpoint('smallTablet')`
+     padding-top: 0;
+    padding-bottom: 0;
+  `};
+`
+
+const AnchorWrapper = ({ to, children }) =>
+  to ? <Anchor to={to}>{children}</Anchor> : children
+
+const ConferenceCard = ({ event }) => {
+  const {
+    status,
+    date,
+    eventName,
+    eventLocation,
+    blurb,
+    homepage,
+    ctaLink,
+    svgImage
+  } = event
+
+  const ctaText = status === 'Upcoming' ? 'Get tickets' : 'Watch on YouTube'
+
+  return (
+    <Row>
+      <Col width={[8 / 12, 8 / 12, 8 / 12, 8 / 12, 0, 0, 0]}>
+        <AnchorWrapper to={homepage}>
+          <Image image={svgImage} width="100%" />
+        </AnchorWrapper>
+      </Col>
+
+      <Col width={[1, 1, 1, 1, 5 / 12, 4 / 12, 4 / 12]}>
+        <StatusWrapper>
+          <BodyPrimary noPadding muted>
+            {status}
+          </BodyPrimary>
+        </StatusWrapper>
+        <CardTitle as="h2">{eventName}</CardTitle>
+        <BodyPrimary>
+          {date} • {eventLocation}{' '}
```

Comment:
I can't see the  `•`  in the design, just a full stop. Has this been updated?

Reply:
Ran it past James on the basis of maintaining consistency between Conference and Event and he said "go for it"

Reply:
ahh cool, just checking!